node/README.md

```diff
@@ -136,3 +136,56 @@ cargo run -- \
 ```
 
 Execute `cargo run -- --help` to learn more about the template node's CLI options.
+
+### Upgrade
+
+To perform a runtime upgrade, Substrate uses existing runtime logic to update 
+the Wasm runtime stored on the blockchain to a new consensus-breaking version with new logic.
+This upgrade gets pushed out to all syncing nodes on the network as a part of the consensus process.
+Once the Wasm runtime is upgraded, the executor will see that the native runtime spec_name, spec_version,
+or authoring_version no longer matches this new Wasm runtime. As a result,
+it will fall back to execute the canonical Wasm runtime instead of using the native runtime
+in any of the execution processes.
+
+Let's do the upgrade step by step:
+1. First make sure your node is running. We are making a runtime upgrade anyway :)
+
+2. Make required changes in code and don't forget to increment spec_verion in RuntimeVersion.
+
+3. Build your new node with
+    ```bash
+    cargo build --release
+    ``` 
+    It will be node state that we will be upgrading to.
+
+    (Old node version is still running)
+
+4. Now use [Polkadot JS Apps UI](https://polkadot.js.org/apps/#/extrinsics?rpc=ws://127.0.0.1:9944) 
+to automatically configure the UI to connect to the local node:
+
+(Some ad blockers and browser restrictions (e.g. the built-in Shield in Brave browser, 
+and https requirement for socket connection in Firefox) interfere with connecting to a local node. 
+Make sure to check them and, if needed, turn them off.
+You may not get connecting from a remote IP (like ```polkadot.js.org/apps/```) to a local node working. 
+If you are unable to solve this, we encourage you to host your app locally, like [the apps UI](https://github.com/polkadot-js/apps#development))
+
+Use related account to invoke the ```sudoUncheckedWeight``` function and use the ```setCode``` function from the
```

Comment:
Please reread Markdown syntax about quoting code: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code

```suggestion
Use related account to invoke the `sudoUncheckedWeight` function and use the `setCode` function from the
```